### ISSUE TITLE
latex image-in-section test fails

### ISSUE THREAD

**Issue by jfbu** [sphinx-doc/sphinx#2667]
steps to reproduce, 

```
make test TEST=test_build_latex.py
cd tests/build/test-image-in-section/_build/latex
make Python.pdf
```

gives

```
! Package keyval Error: WIDTH undefined.
```

This correspond to source

``` rst
test-image-in-section
=====================
this is dummy content


|picture| Test section
----------------------
blah blah blah


Another section
---------------
another blah


Other [blah] |picture| section
------------------------------
other blah

|picture|
---------
blah blah blah
```

with 

``` python
# -*- coding: utf-8 -*-

master_doc = 'index'
exclude_patterns = ['_build']

rst_epilog = '''
.. |picture| image:: pic.png
    :width: 15pt
    :height: 15pt
    :alt: alternative_text
'''
```

as conf.py.

The problem is with things like 

```
\chapter[Test section]{\includegraphics[width=15pt,height=15pt]{{pic}.png} Test section}
```

in latex, because at some point width becomes WIDTH which is not recognized as a valid key. This does not happen without the

```
\usepackage[Bjarne]{fncychap}
```

in the preamble.

**Comment by tk0miya** [comment]
The units for width and heights are not important in the test case.
So please rewrite them to right units like `mm`, `em` and so on.

**Comment by jfbu** [comment]
memo: this test was added at 62c4976 and modified at 7c0b7e1.

**Comment by tk0miya** [comment]
Oh, sorry. I've misread the problem. I reproduced the WIDTH problem at my local.

Note:
It seems `Bjarne` theme calls `\MakeUppercase` to chapter titles. Then the option of includegraphics is capitalized.

**Comment by burnol-math** [comment]
@tk0miya I experimented a fix using ``\lowercase`` but we don't want to do that systematically for all  ``\includegraphics`` only those in chapter titles with sphinxmanual.cls. I did not do a topic branch and I am closing up now for some time on internet. Fix as I recall was to do 

    \lowercase{\includegraphics[options]}

(only the options are wrapped in the ``\lowercase``, not the braces with the filename for inclusion). Rationale is that all option keys of ``\includegraphics`` are lowercase and their values are digits or lowercase already, but I did not have time to test fully. It does make the unit test test-image-in-section work.

**Comment by jfbu** [comment]
@tk0miya I experimented a fix using `\lowercase` but we don't want to do that systematically for all `\includegraphics` only those in chapter titles with sphinxmanual.cls. I did not do a topic branch and I am closing up now for some time on internet. Fix as I recall was to do

```
\lowercase{\includegraphics[options]}
```

(only the options are wrapped in the `\lowercase`, not the braces with the filename for inclusion). Rationale is that all option keys of `\includegraphics` are lowercase and their values are digits or lowercase already, but I did not have time to test fully. It does make the unit test test-image-in-section work.
